src/odemis/driver/spectrometer.py

```diff
@@ -130,7 +130,7 @@ def __init__(self, name, role, dependencies, **kwargs):
 
         # duplicate every other VA and Event from the detector
         # that includes required VAs like .pixelSize and .exposureTime
-        for aname, value in model.getVAs(dt).items() + model.getEvents(dt).items():
+        for aname, value in list(model.getVAs(dt).items()) + list(model.getEvents(dt).items()):
```

Comment:
It might be a little bit more fancy/right to write:
```
from itertools import chain
:
for aname, value in chain(iter(model.getVAs(dt).items()), iter(model.getEvents(dt).items())):
```